include/d/a/d_a_itembase.h

```diff
@@ -0,0 +1,246 @@
+    /* 0x085 */ ITEM_LARGE_QUIVER = 133,
+    /* 0x086 */ ITEM_SMALL_BOMB_BAG = 134,
+    /* 0x087 */ ITEM_MEDIUM_BOMB_BAG = 135,
+    /* 0x088 */ ITEM_LARGE_BOMB_BAG = 136,
+    /* 0x089 */ ITEM_WHIP = 137,
+    /* 0x08A */ ITEM_FIRESHIELD_EARRINGS = 138,
+    /* 0x08B */ ITEM_UNK_139 = 139,
+    /* 0x08C */ ITEM_BIG_BUG_NET = 140,
+    /* 0x08D */ ITEM_FARON_GRASSHOPPER = 141,
+    /* 0x08E */ ITEM_WOODLAND_RHINO_BEETLE = 142,
+    /* 0x08F */ ITEM_DEKU_HORNET_ = 143,
+    /* 0x090 */ ITEM_SKYLOFT_MANTIS = 144,
+    /* 0x091 */ ITEM_VOLCANIC_LADYBUG = 145,
+    /* 0x092 */ ITEM_BLESSED_BUTTERFLY = 146,
+    /* 0x093 */ ITEM_LANAYRU_ANT = 147,
+    /* 0x094 */ ITEM_SAND_CICADA = 148,
+    /* 0x095 */ ITEM_GERUDO_DRAGONFLY = 149,
+    /* 0x096 */ ITEM_ELDIN_ROLLER = 150,
+    /* 0x097 */ ITEM_SKY_STAG_BEETLE = 151,
+    /* 0x098 */ ITEM_STARRY_FIREFLY = 152,
+    /* 0x099 */ ITEM_BOTTLE = 153,
+    /* 0x09A */ ITEM_RUPEE_MEDAL_ALSO = 154,
+    /* 0x09B */ ITEM_HEART_MEDAL_ALSO = 155,
+    /* 0x09C */ ITEM_UNK_156 = 156,
+    /* 0x09D */ ITEM_UNK_157 = 157,
+    /* 0x09E */ ITEM_CAWLINS_LETTER = 158,
+    /* 0x09F */ ITEM_HORNED_COLOSSOL_BEETLE = 159,
+    /* 0x0A0 */ ITEM_BABY_RATTLE = 160,
+    /* 0x0A1 */ ITEM_HORNET_LARVAE = 161,
+    /* 0x0A2 */ ITEM_BIRD_FEATHER = 162,
+    /* 0x0A3 */ ITEM_TUMBLE_WEED = 163,
+    /* 0x0A4 */ ITEM_LIZARD_TAIL = 164,
+    /* 0x0A5 */ ITEM_ELDIN_ORE = 165,
+    /* 0x0A6 */ ITEM_ANCIENT_FLOWER = 166,
+    /* 0x0A7 */ ITEM_AMBER_RELIC = 167,
+    /* 0x0A8 */ ITEM_DUSK_RELIC_AGAIN = 168,
+    /* 0x0A9 */ ITEM_JELLY_BLOB = 169,
+    /* 0x0AA */ ITEM_MONSTER_CLAW = 170,
+    /* 0x0AB */ ITEM_MONSTER_HORN = 171,
+    /* 0x0AC */ ITEM_ORNAMENTAL_SKULL = 172,
+    /* 0x0AD */ ITEM_EVIL_CRYSTAL_ITEM = 173,
+    /* 0x0AE */ ITEM_BLUE_BIRD_FEATHER = 174,
+    /* 0x0AF */ ITEM_GOLDEN_SKULL = 175,
+    /* 0x0B0 */ ITEM_GODDESS_PLUME = 176,
+    /* 0x0B1 */ ITEM_EMERALD_TABLET = 177,
+    /* 0x0B2 */ ITEM_RUBY_TABLET = 178,
+    /* 0x0B3 */ ITEM_AMBER_TABLET = 179,
+    /* 0x0B4 */ ITEM_STONE_OF_TRIALS = 180,
+    /* 0x0B5 */ ITEM_UNK_181 = 181,
+    /* 0x0B6 */ ITEM_UNK_182 = 182,
+    /* 0x0B7 */ ITEM_UNK_183 = 183,
+    /* 0x0B8 */ ITEM_UNK_184 = 184,
+    /* 0x0B9 */ ITEM_UNK_185 = 185,
+    /* 0x0BA */ ITEM_BALLAD_OF_THE_GODDESS = 186,
+    /* 0x0BB */ ITEM_FARORES_COURAGE = 187,
+    /* 0x0BC */ ITEM_NAYRUS_WISDOM = 188,
+    /* 0x0BD */ ITEM_DINS_POWER = 189,
+    /* 0x0BE */ ITEM_FARON_SOTH = 190,
+    /* 0x0BF */ ITEM_ELDIN_SOTH = 191,
+    /* 0x0C0 */ ITEM_LANAYRU_SOTH = 192,
+    /* 0x0C1 */ ITEM_SOTH = 193,
+    /* 0x0C2 */ ITEM_REVITALIZING_POTION_PLUS_PLUS = 194,
+    /* 0x0C3 */ ITEM_HOT_SOUP = 195,
+    /* 0x0C4 */ ITEM_COLD_SOUP = 196,
+    /* 0x0C5 */ ITEM_LIFE_TREE_SEED = 197,
+    /* 0x0C6 */ ITEM_LIFE_TREE_FRUIT = 198,
+    /* 0x0C7 */ ITEM_EXTRA_WALLET = 199,
+    /* 0x0C8 */ ITEM_UNK_200 = 200,
+    /* 0x18D */ ITEM_LIFE_MEDAL_AGAIN = 397,
+    /* 0x18E */ ITEM_POUCH_EXPANSION_GODDESS_CHEST = 398,
```

Comment:
Are these ones actually defined? Seems weird that they'd have 0 -> 200, 397, 398, 511

Reply:
I just copied what was from ghidra, i figured it couldn't hurt to have here. Ill put a comment in